app/(tabs)/profile.tsx

```
@@ -45,37 +47,52 @@ export default function ProfileScreen() {
-        // } else {
-        //   console.log('Update successful:', data);
-        // }
+        // Step 2: Wait for Clerk to sync the new image URL
+        const updatedUser = await user?.reload(); // Ensure we have the latest user data
+        const newImageUrl = updatedUser?.imageUrl;
+
+        // Step 3: Update the `avatar_url` in Supabase
+        if (newImageUrl) {
+          const supabase = createClient(
+            supabaseUrl,
+            process.env.EXPO_PUBLIC_SUPABASE_SERVICE_ROLE_KEY!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)